core/src/themes/md/dark.tokens.ts

```diff
@@ -3,4 +3,68 @@ import type { DarkTheme } from '../themes.interfaces';
 
 export const darkTheme: DarkTheme = {
   ...baseDarkTheme,
+
+  backgroundColor: '#121212',
+  backgroundColorRgb: '18, 18, 18',
+  textColor: '#ffffff',
+  textColorRgb: '255, 255, 255',
+
+  backgroundColorStep: {
+    50: '#1e1e1e',
+    100: '#2a2a2a',
+    150: '#363636',
+    200: '#414141',
+    250: '#4d4d4d',
+    300: '#595959',
+    350: '#656565',
+    400: '#717171',
+    450: '#7d7d7d',
+    500: '#898989',
+    550: '#949494',
+    600: '#a0a0a0',
+    650: '#acacac',
+    700: '#b8b8b8',
+    750: '#c4c4c4',
+    800: '#d0d0d0',
+    850: '#dbdbdb',
+    900: '#e7e7e7',
+    950: '#f3f3f3',
+  },
+
+  textColorStep: {
+    50: '#f33f33',
```

Comment:
So this is weird, but I noticed all of the other colors here seem to be a grey color (`#xyxyxy`) and this one doesn't follow that pattern, so I looked at it and this one is red. Was that intentional?

Maybe you meant `#f3f3f3`? That would be grey like the others.